index.html

```diff
@@ -1563,11 +1583,23 @@ <h2 id="BaseAudioContext">
         </dl>
         <dl title="dictionary AudioContextOptions" class="idl">
           <dt>
-            AudioContextPlaybackCategory playbackCategory = "interactive"
+            (AudioContextLatencyCategory or double) latencyHint = "interactive"
           </dt>
           <dd>
-            Identify the type of playback, which affects tradeoffs between
-            audio output latency and power consumption.
+            <p>
+              Identify the type of playback, which affects tradeoffs between
+              audio output latency and power consumption.
+            </p>
+            <p>
+              The preferred value of the <code>latencyHint</code> is a value
+              from <a>AudioContextLatencyCategory</a>. However, a double can
+              also be specified for the number of seconds of latency for finer
+              control to balance latency and power consumption. It is at the
+              browser's discretion to interpret the number appropriately. The
+              actual latency used is given by AudioContext's <a href=
+              "#widl-BaseAudioContext-baseLatency">baseLatency</a>
+              attribute.
```

Comment:
Is the description of baseLatency in the AudioContext section enough?  Or should I add more text here as well?